backend/log4js-config.json

```diff
@@ -77,15 +77,15 @@
       "type": "stdout",
       "layout": 
       {
-        "type": "pattern", "pattern": "%d{ISO8601} %p %c [%X{user}] [%f : %l] -  %m"
+        "type": "pattern", "pattern": "%d{ISO8601} %p %c [%X{user}] [%f : %l] -  %m %s"
       }
     },
     "apolloOut": 
     {
       "type": "stdout",
       "layout": 
       {
-        "type": "pattern", "pattern": "%d{ISO8601} %p %c [%X{user}] [%f : %l] -  %m"
+        "type": "pattern", "pattern": "%d{ISO8601} %p %c [%X{user}] [%f : %l] -  %m %s"
```

Comment:
i would preferre a defensive solution to enable the stack-output only for error cases.
the layout-pattern for appenders out and appoloOut should not use the %s for stack-output, even when it seems to generate logoutput only on console output. 
this has to be ensured before activating it on master-branch and the log-files will increase in an unexpected manner.